cpp/src/arrow/util/buffer-test.cc

```diff
@@ -53,4 +53,40 @@ TEST_F(TestBuffer, ResizeOOM) {
   ASSERT_RAISES(OutOfMemory, buf.Resize(to_alloc));
 }
 
+TEST_F(TestBuffer, EqualsWithSameContent) {
+	MemoryPool* pool = default_memory_pool();
```

Comment:
It looks like you indenting here is off, which could be one reason why check-format failed.